### DISCUSSION TITLE
Recognize Progress View

### DISCUSSION THREAD

**Discussion by Daddy-ohh** [nextcloud/recognize#558] [Q&A] [answered]
Is there a way to view the Recognize progress towards completion?

I realize the process is going to take a long time, with as many photos as I have, and running on a Raspberry Pi 4.  I am curious though just how much longer it might take.

Fantastic app, by the way!  Kudos for allowing me to retire Photoprism.  Donation coming your way soon.

**Comment by marcelklehr** [comment, marked as answer]
> Is there a way to view the Recognize progress towards completion?

Not currently. It's not easy, because recognize doesn't know how many files are left, only how many are in the queue, and then it doesn't know how many workers you are running, so it'd be a complex algorithm to find out the time left. But it would indeed be a good feature. Can you open a feature request in the issues section?

And, thank you for your donation!

**Reply by Daddy-ohh** [reply]
Feature request added
